bin/setup-matrix

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
Nit: This file probably wants a similar top-level commetn describing that it's intended to dispatch the setup script for every module.